client/client.go

```diff
@@ -63,8 +63,7 @@ func (c *DatabricksClient) GetOAuthToken(ctx context.Context, authDetails string
 
 // Do sends an HTTP request against path.
 func (c *DatabricksClient) Do(ctx context.Context, method, path string,
-	headers map[string]string, request, response any,
-	visitors ...func(*http.Request) error) error {
+	headers map[string]string, request, response any, visitors ...func(*http.Request) error) error {
```

Comment:
Yeah. This `Do()` method should be formulated either as a struct argument or with functional options, since everything after `path` is optional. I'd prefer not to change this signature unless necessary, as it is used in multiple places in the TF provider.

Reply:
Agreed with not using a struct as this is out of scope for this PR. Let's at least format idiomatically then. Our Go style does not mandate a 80 char line length which makes the current formatting quite arbitrary. I'm fine with either having everything on a single line or one argument per line.